lib/tasks/watch_sources.js

```diff
@@ -27,7 +29,7 @@ module.exports = function(gulp, config, watch) {
 		var minimatch = require('minimatch')
 		var path = require('path')
 		var runcmd = require('../helpers/runcmd')
-		var glob = '**/*.{css,styl,scss,less,js,jsx,es6,es,coffee,jade,json,html,htm}'
+		var glob = '**/*{' + fileExtensions.join() + "}"
```

Comment:
"extension of hardcoded types" means that it would watch only file extension in default mapping?